ui/README.md

```diff
@@ -85,6 +83,12 @@ the conversation and make changes you think would be worthwhile.
    the form is submitted. This helps demonstrate the prior requirement (that the
    default value is added).
 1. Add a button that allows the builder to clear the form and start fresh.
+1. The submit button should create a json object and post it to a
```

Comment:
Please remove "a". It should read "The submit button should create a json object and post it to http://www.mocky.io/v2/566061f21200008e3aabd919."